VDF.Core/FFTools/FfmpegEngine.cs

```diff
@@ -29,7 +29,9 @@ static class FfmpegEngine {
 		public static byte[]? GetThumbnail(FfmpegSettings settings) {
 			using var process = new Process {
 				StartInfo = new ProcessStartInfo {
-					Arguments = $" -hide_banner -loglevel panic -y {(UseCuda ? "-hwaccel cuda" : string.Empty)} -ss {settings.Position} -i \"{settings.File}\" -t 1 -f {(settings.GrayScale == 1 ? "rawvideo -pix_fmt gray" : "mjpeg")} -vframes 1 {(settings.GrayScale == 1 ? "-s 16x16" : "-vf scale=100:-1")} \"-\"",
+					Arguments = $" -hide_banner -loglevel panic -y {(UseCuda ? "-hwaccel cuda" : string.Empty)} -ss {settings.Position} -i \"{settings.File}\" -t 1 " +
+					            $"-f {(settings.GrayScale == 1 ? "rawvideo -pix_fmt gray" : "mjpeg")} -vframes 1 " +
+								$"{(settings.GrayScale == 1 ? $"-s {settings.Width}x{settings.Width}" : $"-vf scale={settings.Width}:-1")} \"-\"",
```

Comment:
Either use @ for multi line or make it single line again.

Reply:
OK.
But what is bad about splitting a very long string by using `+` in order to make it readable?

Reply:
I'm not sure if the compiler merges it into one string or if its getting joined at runtime, if you use `+`, which costs performance.

Reply:
again one line